mypy/checkexpr.py

```diff
@@ -90,7 +90,7 @@
 MAX_UNIONS = 5  # type: Final
 
 
-# Types considered safe for comparisons with --strict-equality due to known behaviour of __eq__.
+# Types considered safe for comparisons with --strict-equality due to known behavior of __eq__.
```

Comment:
mypy seems to use a mix of non-American and American English, so could take or leave this one